### PR TITLE
gives the mosin, paladin 12, and ppsh new sprites

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
This is literally just splinters pr but with a shoter mosin and no balance changes

and a new paladin 12 I guess.

## Why It's Good For The Game
The old mosin was really bad and isn't an excuse to make the new one also really long.

## Changelog
:cl:
MJP, Splinter
add: Added new mosin,ppsh, and paladin 12 sprites by ocelot. PPSH and Mosin added by Splinter
del: Removed old paladin and mosin and ppsh sprites, git will remember them. Nobody else will.
:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -163. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by SplinterGP** [comment]
MJP you removed a irremoveable barrel from mosin and it had stats, add it to gun ree

File: code/modules/projectiles/guns/shotguns.dm [hunk 1]
```diff
@@ -641,9 +641,9 @@ can cause issues with ammo types getting mixed up during the burst.
 	aim_fire_delay = 1
 
 	fire_delay = 17.5
-	accuracy_mult = 1.4
+	accuracy_mult = 1.45
```
Comment (Pariah919) [inline code/modules/projectiles/guns/shotguns.dm:644]: had an irremoveable barrel with stats for some reason, nice one high, same for the scatter

**Review by TheBimmer231 — APPROVED** [review]

File: code/modules/projectiles/guns/shotguns.dm (hunk 1 above)
Comment (SplinterGP) [reply code/modules/projectiles/guns/shotguns.dm:644]: same happens to stock as well

**Review by SplinterGP — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-189). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**